__tests__/cairo1.test.ts

```diff
@@ -550,8 +550,8 @@ describeIfDevnet('Cairo 1 Devnet', () => {
   });
 });
 
-describeIfSequencerGoerli('Cairo1 Testnet', () => {
-  describe('Sequencer API - C1 Testnet C:0x00305e...', () => {
+describeIfCairo1Testnet('Cairo1 Testnet', () => {
```

Comment:
why Cairo1Testnet? 
I think it can just be `describeIfTestnet`
cc @tabaktoni

Reply:
`describeIfTestnet` lgtm!